src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+:root {
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo__img {
+  display: flex;
+  height: 40px;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0;
+  padding-left: 0;
+}
+
+.nav__item {
+  display: flex;
+  text-align: center;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  position: relative;
+  font-weight: 500;
+  font-size: 12px;
+  height: 60px;
+  line-height: 60px;
+  align-items: center;
+  text-transform: uppercase;
+  margin-left: 20px;
```

Comment:
Be careful with adding both left and right margins to elements as it can lead to unexpected layout behaviors due to margin collapsing. In this case, '.nav__link' has a 'margin-left', which is fine, but ensure that you don't add a 'margin-right' to the same element.